ra/ra.go

```diff
@@ -600,14 +599,14 @@ func (ra *RegistrationAuthorityImpl) checkPendingAuthorizationLimit(ctx context.
 	if err != nil {
 		return err
 	}
-	if overrideKey != "" {
-		utilization := float64(countPB.Count) / float64(threshold)
-		ra.rlOverrideUsageGauge.WithLabelValues(ratelimit.PendingAuthorizationsPerAccount, overrideKey).Set(utilization)
-	}
 	if countPB.Count >= threshold {
 		ra.log.Infof("Rate limit exceeded, PendingAuthorizationsByRegID, regID: %d", regID)
 		return berrors.RateLimitError(0, "too many currently pending authorizations: %d", countPB.Count)
 	}
+	if overrideKey != "" {
+		utilization := float64(countPB.Count+1) / float64(threshold)
```

Comment:
I think this +1 is not correct: the NewOrder request which spawned this rate limit check might be for a cert with 100 names and therefore 100 new pending authorizations. But making it correct would require some additional unfortunate plumbing =/